composer.json

```diff
@@ -23,10 +23,10 @@
         "phpseclib/phpseclib": "^3.0",
         "phpseclib/phpseclib2_compat": "^1.0",
         "spomky-labs/php-aes-gcm": "^1.2",
-        "symfony/process": "^4.1|^5.0"
+        "symfony/process": "^5.0|^6.0"
```

Comment:
I think we should keep support for symfony/process 4 as Symfony 4.4 is a LTS and support will end only in November 2023.

Reply:
The current LTS is 5.4 and very soon 6.4 will be available, so we can safely switch to the proposed versions.

Reply:
There can be multiple LTS at one time. Yes, Symfony 4 LTS is not supported anymore but not a reason to stop supporting it if there is no maintenance effort required.
As far as I can see, this PR should be merged, but we should add first automated tests using github actions, replacing old unsupported travis, as I'm not sure if it works perfectly.